docs/lazy-adr/adr-004-mvp-light-client.md

```diff
@@ -0,0 +1,273 @@
+# ADR 004: Data Availability Sampling light-client
```

Comment:
```suggestion
# LAZY ADR 004: Data Availability Sampling Light Client
```

Reply:
I'd actually prefer to remove the LAZY prefix even from previous ADRs. It doesn't add any value. It made sense when we kept the other (tendermint) ADRs around as well but not anymore. I'll remove it in the other existing adrs too.

Reply:
https://github.com/lazyledger/lazyledger-core/pull/311/commits/eb64c8e0ae7c5867e92b4f899af13993175c4e0b